com.unity.render-pipelines.core/Editor/Volume/VolumeComponentListEditor.cs

```diff
@@ -308,7 +316,8 @@ void OnContextClick(Vector2 position, VolumeComponentEditor targetEditor, int id
                 menu.AddItem(EditorGUIUtility.TrTextContent("Show Additional Properties"), targetEditor.showAdditionalProperties, () => targetEditor.showAdditionalProperties ^= true);
             else
                 menu.AddDisabledItem(EditorGUIUtility.TrTextContent("Show Additional Properties"));
-            menu.AddItem(EditorGUIUtility.TrTextContent("Show All Additional Properties..."), false, () => CoreRenderPipelinePreferences.Open());
+            menu.AddItem(EditorGUIUtility.TrTextContent("Show Only Overrided Parameters"), targetEditor.showOnlyOverridedParameters, () => targetEditor.showOnlyOverridedParameters = !targetEditor.showOnlyOverridedParameters);
```

Comment:
Proper form is "overridden", not "overrided". Replace at least user-facing text, preferably also variable names.